cluster/log-dump/log-dump.sh

```diff
@@ -438,6 +444,10 @@ function dump_nodes() {
     if [[ proc -eq 0 ]]; then
       proc=${max_dump_processes}
       wait
+      now="$(date +%s)"
+      if [[ -n "${log_dump_ssh_timeout_seconds}" && $((now - start)) -gt ${log_dump_ssh_timeout_seconds} ]]; then
+        break
```

Comment:
Let's "log" (echo?) that we hit the timeout and we'll be finishing shortly.

Reply:
Done.